articles/azure-functions/functions-dotnet-dependency-injection.md

```diff
@@ -71,6 +71,8 @@ A series of registration steps run before and after the runtime processes the st
 
 - *The dependency injection container only holds explicitly registered types*. The only services available as injectable types are what are setup in the `Configure` method. As a result, Functions-specific types like `BindingContext` and `ExecutionContext` aren't available during setup or as injectable types.
 
+- *Configuring ASP.NET authentication is not supported*. The Functions host configures ASP.NET authentication services in order to properly expose APIs to infrastucture services, such as the Azure portal. Additional configuration in a custom Startup class can override this configuration, causing unintended consequences. For example, calling `builder.Services.AddAuthentication()` can break authentication between the portal and the host, leading to messages such as "Azure Functions runtime is unreachable".
```

Comment:
The consequences here are broader than just portal scenarios, right? Any non-HttpTrigger endpoint would potentially be in scope, including extension endpoints, etc., right? Regardless, we might reframe that more as a specific example. 

```suggestion
- *Configuring ASP.NET authentication is not supported*. The Functions host configures ASP.NET authentication services in order to properly expose APIs for core lifecycle operations. Additional configuration in a custom Startup class can override this configuration, causing unintended consequences. For example, calling `builder.Services.AddAuthentication()` can break authentication between the portal and the host, leading to messages such as "Azure Functions runtime is unreachable".
```